dist/index.js

```diff
@@ -22674,44 +22674,31 @@ async function getLatestVolta(authToken) {
 function voltaVersionHasSetup(version) {
     return semver.gte(version, '0.7.0');
 }
-async function buildDownloadUrl(platform, version, openSSLVersion = '') {
-    let fileName;
-    switch (platform) {
-        case 'darwin':
-            fileName = `volta-${version}-macos.tar.gz`;
-            break;
-        case 'linux': {
-            openSSLVersion = await getOpenSSLVersion(openSSLVersion);
-            fileName = `volta-${version}-linux-${openSSLVersion}.tar.gz`;
-            break;
+async function buildDownloadUrl(platform, version, variant = '', openSSLVersionForTesting = '') {
```

Comment:
Let's reset the `dist/` changes in the PR. I prefer to update `dist/` when I do tags as part of the release process. It makes reviewing things a bit easier in the PR diff.

Reply:
Yep I can. I built to make sure there were no compiler errors.

Reply:
Done